sunbeam-python/sunbeam/commands/hypervisor.py

```diff
@@ -153,6 +153,14 @@ def __init__(
             "Adding Openstack Hypervisor unit to machine(s)",
         )
 
+    def get_accepted_unit_status(self) -> dict[str, list[str]]:
```

Comment:
The fix should be extended to ReapplyHypervisorTerraformPlanStep
[1] https://github.com/canonical/snap-openstack/blob/main/sunbeam-python/sunbeam/commands/hypervisor.py#L325